src/main/java/school/lemon/changerequest/java/Employees/Accountant.java

```diff
@@ -0,0 +1,24 @@
+package school.lemon.changerequest.java.Employees;
+
+/**
+ * Created by User on 05.12.2016.
+ */
+public class Accountant extends Employees {
+    private static int countAccountant;
+    public Accountant(String nameE, int workHoursE) {
+        super(nameE, workHoursE);
+        countAccountant++;
+        Manager.SalaryOverAll=Manager.SalaryOverAll+getSalary();
```

Comment:
Accountant should contain array of employees and calculate their salary instead of static fields. 
You should avoid static fields/methods everywhere it possible.